### ISSUE TITLE
Feature suggestion: when I give wrong answer, show details of my answer

### ISSUE THREAD

**Issue by kierenj** [davidsansome/tsurukame#174]
E.g. I was tested on the colour radical. I said Bed.
In many cases I’m not sure where I went wrong, do I open a browser, hit WaniKani and search Bed and flick back to the app to see the difference.

It might work great to show what my answer relates to, so the difference is obvious (“ah, I can see where I went wrong - this one has a .. whatever”).

What do you think?

**Comment by turtlemaster19** [comment]
Interesting. How would this work? Would we try and see if there is an item with that name and if so, display it? What about if you had typed "Bad" or something?

**Comment by kierenj** [comment]
Yes that's what I was thinking! A reverse-search. Sometimes I'm just so sure of my (wrong) answer, I feel compelled to understand where I went wrong, and to do so I need to see them side by side - closest I can get is opening a tab in Safari to look up my answer.

If I'd typed bad I guess it could show any matches for 'Bad', and I'd probably ignore it, thinking oops I meant 'bed'.

Another way to make it easier would be to add a lookup feature like the main WaniKani site, within the app, so upon making a wrong guess I could use that without having to leave it. But I think giving info about the wrong answer inline as it happens to me feels super useful.
